components/homme/src/theta-l_kokkos/cxx/HyperviscosityFunctorImpl.hpp

```diff
@@ -175,7 +211,45 @@ class HyperviscosityFunctorImpl
     m_sphere_ops.vlaplace_sphere_wk_contra(kv, m_data.nu_ratio1,
                               Homme::subview(m_state.m_v,kv.ie,m_data.np1),
                               Homme::subview(m_buffers.vtens,kv.ie));
-  }
+
+    Kokkos::parallel_for(Kokkos::TeamThreadRange(kv.team,NP*NP),
```

Comment:
I was gonna say that we need a team barrier before this TTR for too, but turns out that all our sph ops have a team barrier at the end. I would argue that we should remove the team barrier at the end of our sph ops (the barrier may not be necessary until later on), but it should not be part of this PR.